packages/batch/src/index.ts

```diff
@@ -28,6 +35,7 @@ export function batch(opts?: Partial<BatchOptions>) {
     async function consume() {
       const pending = operations;
       const body = `[${operations.map(o => o.body).join(',')}]`;
+      const fetchOpts = mergeFetchOpts(opContext, { headers: {}, body });
```

Comment:
Looks alright but do we need to pass the empty `headers` object on the right-hand side? Let's omit it if possible.

Reply:
If I did not pass the empty headers object I got a type error which said a headers object was expected for the FetchOptions type.

Reply:
Right, I think that's fine for now. I will merge this and add a test for it.

Reply:
Sounds good!